test/integration/targets/azure_rm_routetable/aliases

```diff
@@ -0,0 +1,3 @@
+cloud/azure
+posix/ci/cloud/group2/azure
```

Comment:
Some of the aliases were recently change, have a look at https://github.com/ansible/ansible/blob/devel/test/integration/targets/azure_rm_loadbalancer/aliases to see the new alias name.